ACE/ace/Handle_Set.h

```diff
@@ -71,6 +71,20 @@ class ACE_Export ACE_Handle_Set
    */
   ACE_Handle_Set (const fd_set &mask);
 
+
```

Comment:
Did something change in Handle_Set.h/inl, looking at the diffs it only looks some operations got moved in the file itself?